docs/docusaurus.config.ts

```diff
@@ -27,6 +27,13 @@ const config: Config = {
           includeCurrentVersion: true,
           lastVersion: '0.1.0',
         },
+        blog: {
+          showReadingTime: true,
+          // Please change this to your repo.
```

Comment:
Is it left-over?

Reply:
[6755847](https://github.com/zkmopro/mopro/pull/288/commits/675584747e16b857c3f8e6d710c339e2faa7005e)